client/src/components/Request.js

```diff
@@ -171,20 +171,23 @@ const Request = () => {
     if (reviewId) {
       handleInitState();
       socket.messages(reviewId);
-      console.log(reviewId);
     }
     // TODO remove request when unmounting
   }, [reviewId]);
 
   const handleSocketMessage = (message) => {
     const { newMessage, reviewOwner, selectedReviewer } = message;
-    dispatch({
-      type: "NEW_MESSAGE",
-      review: newMessage.embeddedReview,
-      selectedReviewer: selectedReviewer,
-      reviewOwner: reviewOwner,
-      status: newMessage.status,
-    });
+    const currentReview = window.location.href.slice(-24);
```

Comment:
There should be another way. It's a bad practice to use window.location.href in single page app

Reply:
Replaced with React's useParam().